src/resources/breaking-changes.md

```diff
@@ -0,0 +1,828 @@
+---
+title: Breaking changes
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists the breaking changes in Dart, organized by release and area.
+Complete release notes are available in the [Dart SDK change log][change log].
+
+According to our [breaking change policy], each listed change contains a link to more
+information on the change, such as migration guidance, background, reasoning,
+caveats, and more.
+
+{{site.alert.note}}
+Changes prior to Dart 3.1.0 and our updated breaking change policy 
+might not have links to additional information. More details on some of those
+changes might be available in their corresponding [change log] entry.
+{{site.alert.end}}
+
+**Dart has two types of breaking changes**:
+* **Standard**: Causes breaks as soon as you [upgrade your sdk version][sdk]. 
+  *These are the majority of changes and are not specially marked in this list.*
+* **Language versioned**: Causes breaks only when you [upgrade the
```

Comment:
If we keep this labeling mechanism, I think it would be worth going through past versions (perhaps to Dart 2.7 when language versioning was released) and labeling language versioned changes, so we can be consistent throughout. At least to 2.12 since those language versions are still supported.

Reply:
I'm _pretty sure_ I labeled all the ones that were explicitly called out already in their original changelog entries. I could've missed some though, but I'd need someone else to confirm because I have no way of knowing unless they're labeled